.github/workflows/docker.yml

```diff
@@ -1,8 +1,9 @@
 name: "Docker Build"
 on:
+  # build a daily dev image  
+  schedule:
+    - cron: "6 42 * * *"
```

Comment:
Think we need to swap minute/hour here

```suggestion
    - cron: "42 6 * * *"
```

Reply:
doh, thanks!

Reply:
also noting actionlint did find this issue, re future listing action.